components/gameComponents/hand.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import styles from "@/app/page.module.css";
```

Comment:
i'm victim to this too, but we should be consistent with our single vs double quote usage. perhaps some sort of linter would benefit us here too.

```suggestion
import styles from '@/app/page.module.css';
```